### DISCUSSION TITLE
app crashed

### DISCUSSION THREAD

**Discussion by ironcatandorange** [NoteProtocol/protocol#2] [General]
after mint a note I always got app crashed.
how can I fix it?
![image](https://github.com/NoteProtocol/protocol/assets/89032826/a6ef472d-b9fc-433c-9dd9-773ee74c34e5)

**Comment by baryon** [comment]
The Urchain BTC data server is temporarily suspended due to high traffic and will resume service as soon as possible after being repaired next week. It will not be available over the weekend.

**Reply by Bradbuythedip** [reply]
approx ETA?
